.github/workflows/fuzzer.yml

```diff
@@ -0,0 +1,21 @@
+name: "Fuzzer Workflow"
+on:
+  schedule:
+  # At the end of every day
+  - cron: "0 0 * * *"    
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y build-essential
+
+    - name: Run fuzzer
+        run: |
+          cd fuzzer
+          cargo hfuzz run fuzzer
```

Comment:
Idk if this is necessary but this is the command to limit how long the fuzzer runs. It could be chained with corpus minimization and such
--run_time VALUE
	Number of seconds this fuzzing session will last (default: 0 [no limit])

Reply:
yes!!! it will be updated later, is a work in progress yet